src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -291,149 +294,160 @@ private void removeAttribute(org.w3c.dom.Element element, String input, String a
   /**
    * Create a MessageML element based on the DOM element's name and attributes.
    */
-  public Element createElement(org.w3c.dom.Element element, Element parent) throws
+  public Optional<Element> createElement(org.w3c.dom.Element element, Element parent) throws
```

Comment:
I'll raise the "backward compatibility" flag :) As this library is open-source, we can't really know whom is using it actually. 

Is it really necessary to modify the contract for this method? Can't we just return `null` in case of invalid `LabelableElement.LABEL` Element?

Reply:
You are right @symphony-thibault I changed it. However please note that backward compatibility could be broken anyway, because the original method never returns null.